recipes/ngmix/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "ngmix" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/esheldon/ngmix.git
```

Comment:
There most likely should have been some discussion about this, we usually only accept tarball http://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=tarballs#tarballs-no-repos